From f0035e8f335f61683cacc2b622e8ed992b6bea0b Mon Sep 17 00:00:00 2001
From: Michael Kuhn <github@ikkoku.de>
Date: Sat, 16 Sep 2023 19:54:48 +0200
Subject: [PATCH] hwdb: Add Logitech G502 X

The already existing entry refers to the Logitech G502 X LIGHTSPEED,
rename it accordingly.

DPI and frequency were taken from Logitech G HUB and
https://www.logitech.com/assets/66240/2/g502-x-corded-gaming-mice-artanis.pdf

(cherry picked from commit 4eebc002b00076048af5e3cbafa1eb0d6319a928)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 46d723a92e..7d1370ea9b 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -397,9 +397,13 @@ mouse:usb:v046dpc08b:name:Logitech G502 HERO SE:*
 mouse:usb:v046dpc08b:name:Logitech G502 HERO Gaming Mouse:*
  MOUSE_DPI=1200@1000 *2400@1000 3200@1000 6400@1000
 
-# Logitech G502 X (Wired)
+# Logitech G502 X
+mouse:usb:v046dpc099:name:Logitech G502 X:*
+ MOUSE_DPI=800@1000 1200@1000 *1600@1000 2400@1000 3200@1000
+
+# Logitech G502 X LIGHTSPEED (Wired)
 mouse:usb:v046dpc098:name:Logitech, Inc. G502 X LIGHTSPEED:*
-# Logitech G502 X (Wireless)
+# Logitech G502 X LIGHTSPEED (Wireless)
 # The USB receiver is also used by other mice. See #27118.
 # If you want to enable the entry, please copy below to your custom hwdb file.
 #mouse:usb:v046dpc547:name:Logitech USB Receiver:*
